### PR TITLE
Mentor home

### PR DESCRIPTION
**Pull request by harrisonliew**
https://github.com/ucb-bar/hammer-cad-plugins/issues/41 in order to make https://github.com/ucb-bar/hammer-cad-plugins/pull/40 work

### CODE REVIEW

**Comment by edwardcwang** [comment]
@harrisonliew I almost forgot, can you add similar contraption to MentorTool like https://github.com/ucb-bar/hammer/blob/e6ced0dae5da0a2d8b19ce2edd4d8f4b1bbf8c61/src/hammer-vlsi/hammer_vlsi/hammer_vlsi_impl.py#L845 ?

File: src/hammer-vlsi/hammer_vlsi/hammer_vlsi_impl.py [hunk 1]
```diff
@@ -1170,17 +1170,33 @@ def get_synopsys_rm_tarball(self, product: str, settings_key: str = "") -> str:
...
+            "MGLS_LICENSE_FILE": self.get_setting("mentor.MGLS_LICENSE_FILE"),
+            "MENTOR_HOME": self.get_setting("mentor.mentor_home")
+        }
+
+        def update_dict(old: dict, new: dict) -> dict:
+            tmp = deepdict(old)
+            tmp.update(new)
+            return tmp
+
+        return reduce(update_dict, [dict(super().env_vars)] + list_of_vars + [mentor_vars], {})
```
Comment (edwardcwang) [inline src/hammer-vlsi/hammer_vlsi/hammer_vlsi_impl.py]: Can we use `add_dicts` in hammer_utils? See #327

**Comment by jwright6323** [comment]
@harrisonliew can we squash these?

**Comment by harrisonliew** [comment]
Sure, go ahead

**Pull request merged**